### PR TITLE
Improved GetRelevantEvents for User

### PR DESCRIPTION
**Pull request by israeltq**
Based on other teams code review, improved and optimized our logic for retrieving the best events for users.

### CODE REVIEW

File: api/handlers/event.js [hunk 1]
```diff
@@ -320,33 +320,24 @@ let searchEvents = async function(req, res) {
 
 // TODO: Define a default limit of returned events
```
Comment (adrianosela) [inline api/handlers/event.js]: remove this stub, you're already defining  default limit

Comment (israeltq) [reply api/handlers/event.js]: Oh, but its temporary. I meant like a more appropriate one. For now I just left it at ten but maybe we want to return more. What do you think?

Comment (adrianosela) [reply api/handlers/event.js]: yeah, lets set it to 50 and remove the stub

Comment (israeltq) [reply api/handlers/event.js]: Will do. Thanks

**Pull request merged**